SWEN766BetterMaps/app/src/test/java/com/example/swen766_bettermaps/LoginTest.java

```diff
@@ -0,0 +1,10 @@
+package com.example.swen766_bettermaps;
+
+import org.junit.Test;
+
+public class LoginTest {
+    @Test
+    public void sumTest() {
```

Comment:
maybe remove this redundant test? Or add a // TODO comment for a different test to be implemented

Reply:
Added TODO comments.